chains/container-chains/runtime-templates/frontier/src/lib.rs

```diff
@@ -1148,9 +1153,13 @@ impl_runtime_apis! {
                         }
                     };
 
+                    // TODO: call_weight or call_weight+extension_weight
+                    // In stable2412, polkadot has added extension weight, which is an extra weight
+                    // for some transaction validations such as nonce, age, signature, etc.
+                    // Currently we ignore that in the frontier template
```

Comment:
Is there any specific reason you want to ignore it?

Reply:
To keep the previous weights where possible, we should change it but I assume it will break many tests, so for a future PR